tests/functional/pv/pv_services/test_cephfs_pvc_with_chunk_size.py

```diff
@@ -0,0 +1,96 @@
+import time
+import pytest
+import logging
+from ocs_ci.framework.testlib import bugzilla, tier2
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import pod
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import cluster
+from ocs_ci.framework.pytest_customization.marks import magenta_squad
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    argnames="interface",
+    argvalues=[
+        pytest.param(
+            *[constants.CEPHFILESYSTEM], marks=pytest.mark.polarion_id("OCS-323")
+        ),
+    ],
+)
+@tier2
+@bugzilla("2171225")
+@magenta_squad
+class TestCephfsWithChunkIo:
+    """
+    This class takes care of create Cephfs PVC, create Fedora dc pod and run Chunk IO on fedora pod
+    """
+
+    def test_cephfs_with_large_chunk_io(self, interface, pvc_factory, dc_pod_factory):
+
+        """
+        This function facilitates
+        1. Create PVC with Cephfs, access mode RWX
```

Comment:
RWX is a must for this issue?

Reply:
Customer hit this issue in cephfs and BZ also filed on CephFs. No information provided about specific access mode. So removed it and choose default one.